docs/api.md

```diff
@@ -366,6 +366,13 @@ connections, nor does it pause the streams of existing connections or their wire
 
 Resume connecting to new peers.
 
+## `torrent.verifyPieces([function callback (err) {}])`
+
+Verify the hashes of all pieces in the store. Useful if the store has been modified
+outside WebTorrent. If a piece hash is incorrect, the torrent will be destroyed, and
```

Comment:
If a piece hash is incorrect, nothing happens. The underlying `_verifyPieces` method just marks the valid pieces as verified and ignores the invalid pieces. So I think this should say that the callback is called when verification is complete, and an error is returned if the torrent is destroyed during verification for any reason. But, `null` will be returned if verification completes successfully, whether or not all the pieces hash correctly or not.

Reply:
Yes, you're right. I made this change a while ago and didn't look at it closely enough again.